state_machine/states/impl/odlc_impl.py

```diff
@@ -23,9 +106,102 @@ async def run(self: ODLC) -> State:
     """
     try:
         logging.info("ODLC")
-        # code for the ODLC logic
 
-        return Airdrop(self.drone, self.flight_settings)
+        # Initialize the camera
+        camera: Camera = Camera()
+
+        # These waypoint values are all that are needed to traverse the whole odlc drop location
+        # because it is a small rectangle
+        # The first waypoint is the midpoint of
+        # the left side of the rectangle(one of the short sides), the second point is the
+        # midpoint of the right side of the rectangle(other short side),
+        # and the third point is the top left corner of the rectangle
+        # it goes there for knowing where the drone ends to travel to each of the drop locations,
+        # the altitude is locked at 100 because
+        # we want the drone to stay level and the camera to view the whole odlc boundary
+        # the altitude 100 feet was chosen to cover the whole odlc boundary
+        # because the boundary is 70ft by 360ft the fov of the camera
+        # is vertical 52.1 degrees and horizontal 72.5,
+        # so using the minimum length side of the photo the coverage would be 90 feet allowing
+        # 10 feet overlap on both sides
+        waypoint: dict[str, list[float]] = {
+            "lats": [38.31451966813249, 38.31430872867596, 38.31461622313521],
+            "longs": [-76.54519982319357, -76.54397320409971, -76.54516993186949],
+            "Altitude": [100],
+        }
+
+        # traverses the 3 waypoints starting at the midpoint on left to midpoint on the right
+        # then to the top left corner at the rectangle
+        with open("flight/data/output.json", encoding="ascii") as output:
+            airdrop_dict = json.load(output)
+            airdrops: int = len(airdrop_dict)
+            point: int
+        while airdrops != 5:
+            logging.info("Starting odlc zone flyover")
+
+            for point in range(3):
+                take_photos: bool = False
+
+                if point == 0:
+                    logging.info("Moving to the center of the west boundary")
+                elif point == 1:
+                    # starts taking photos at a .5 second interval because we want
+                    # to get multiple photos of the boundary so there is overlap and
+                    # the speed of the drone should be 20 m/s which is 64 feet/s which means
```

Comment:
We'll have to talk some more, but others are concerned that this speed may cause blurry photos, so this is something to hash out in testing.